### PR TITLE
feat: WDS `bgNeutral`, `fgNeutral`

### LINKED ISSUE #25060: feat: WDS `Neutral` color algo

### PR DESCRIPTION
**Pull request by ichik**
## Description

tl;dr Automatic neutrals for arbitrary seed.

Fixes #25060 

## Type of change
- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- Manual

### Test Plan
Initial POC refinement, no testing necessary

### Dev activity
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

File: app/client/packages/design-system/theming/src/color/DarkModeTheme.ts [hunk 1]
```diff
@@ -433,8 +433,32 @@ export class DarkModeTheme implements ColorModeTheme {
     return color;
   }
 
+  // Low chroma, but not 0, if possible, to produce harmony with accents in the UI
   private get bgNeutral() {
-    return "#f5f7fa";
+    const color = this.bgAccent.clone();
+
+    // For darker accents it helps to increase neutral's lightness a little, so it's visible against bg
```
Comment (KelvinOm) [inline app/client/packages/design-system/theming/src/color/DarkModeTheme.ts:440]: Is this the expected result?

Снимок экрана 2023-07-11 в 18 41 51

**Review by KelvinOm — CHANGES_REQUESTED** [review]

File: app/client/packages/design-system/theming/src/color/DarkModeTheme.ts (hunk 1 above)
Comment (ichik) [reply app/client/packages/design-system/theming/src/color/DarkModeTheme.ts:440]: Yes, `on*` colors will be taken care of in #25018

**Review by KelvinOm — APPROVED** [review]
